crypto/arm_arch.h

```diff
@@ -115,6 +116,8 @@ extern unsigned int OPENSSL_armv8_rsa_neonized;
 # define ARM_CPU_PART_N3           0xD8E
 # define ARM_CPU_PART_V3           0xD84
 
+#define QCOM_CPU_PART_ORYON_X1     0x001
+
```

Comment:
style checker wants this as `# define` similar to all the ones above it.

Reply:
Fixed